packages/contracts/src/utils/RelayHubValidator.sol

```diff
@@ -26,7 +26,8 @@ library RelayHubValidator {
             dynamicParamSize(approvalData) +
             dynamicParamSize(relayRequest.request.data) +
             dynamicParamSize(relayRequest.relayData.paymasterData);
-        require(signature.length <= 65, "invalid signature length");
+        // zero-length signature is allowed in a batch relay transaction
+        require(signature.length <= 65 || signature.length == 0, "invalid signature length");
```

Comment:
not needed: zero is a special case of <= 65...

Reply:
remove zero-check. it is covered by signature.length <=

Reply:
Fixed.